src/plugins/intel_gpu/src/kernel_selector/cl_kernels/ctc_greedy_decoder_ref.cl

```diff
@@ -7,8 +7,11 @@
 KERNEL(ctc_greedy_decoder_ref)(const __global INPUT0_TYPE* probabilities
                               ,const __global INPUT1_TYPE* sequence_indicators
                                     ,__global OUTPUT_TYPE* output_sequences
-#ifdef SECOND_OUTPUT_EXIST
+#ifdef LEGACY_MULTIPLE_OUTPUTS
                                     ,__global INPUT2_TYPE* second_output
+#endif
+#ifdef NEW_MULTIPLE_OUTPUTS
```

Comment:
Why don't you use #else if? Is there any case that both LEGACY and NEW are defined at the same time?

Reply:
No, but fixed code to use #elif to make it clear. Thanks for your comments.